terraform/modules/service/variables.tf

```diff
@@ -0,0 +1,153 @@
+  type = list(object({
+    name  = string
+    value = string
+  }))
+  default = null
+}
+
+variable "container_secrets" {
+  description = "The secrets to pass to the container. For more information, see [Specifying Sensitive Data](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/specifying-sensitive-data.html) in the Amazon Elastic Container Service Developer Guide"
+  type = list(object({
+    name      = string
+    valueFrom = string
+  }))
+  default = null
+}
+
+variable "cpu" {
+  description = "Number of cpu units used by the task."
+  type        = number
+}
+
+variable "desired_count" {
+  description = "Number of instances of the task definition to place and keep running."
+  type        = number
+  default     = 0
+}
+
+variable "family_name_override" {
+  default     = null
+  description = "The desired family name for the ECS task definition.  If null will default to: {var.platform.env}-{var.platform.app}-{var.platform.service}"
+}
+
+variable "force_new_deployment" {
+  default     = false
+  description = "When *changed* to `true`, trigger a new deployment of the ECS Service even when a deployment wouldn't otherwise be triggered by other changes. **Note**: This has no effect when the value is `false`, changed to `false`, or set to `true` between consecutive applies."
+  type        = bool
+}
+
+
+variable "image" {
+  description = "The image used to start a container. This string is passed directly to the Docker daemon. By default, images in the Docker Hub registry are available. Other repositories are specified with either `repository-url/image:tag` or `repository-url/image@digest`"
+  type        = string
+  default     = null
+}
+
+variable "load_balancers" {
+  description = "Load balancer(s) for use by the AWS ECS service."
+  type = list(object({
+    target_group_arn = string
+    container_name   = string
+    container_port   = number
+  }))
+}
+
+variable "memory" {
+  description = "Amount (in MiB) of memory used by the task."
+  type        = number
+}
+
+variable "mount_points" {
+  description = "The mount points for data volumes in your container"
+  type = list(object({
+    containerPath = optional(string)
+    readOnly      = optional(bool)
+    sourceVolume  = optional(string)
+  }))
+  default = [
+    {
+      containerPath = "/mnt/efs"
+      sourceVolume  = "efs"
+      readOnly      = true
+    },
+    {
+      containerPath = "/tmp"
+      sourceVolume  = "tmp"
+      readOnly      = false
+    },
+    {
+      containerPath = "/var/log"
+      sourceVolume  = "var_log"
+      readOnly      = false
+    }
+  ]
+}
+
+
+variable "network_configurations" {
+  description = "Network configuration for the aws ecs service."
+  type = list(object({
+    subnets          = list(string)
+    assign_public_ip = string
+    security_groups  = list(string)
+  }))
+}
+
+variable "platform" {
+  description = "Object that describes standardized platform values."
+  type        = any
```

Comment:
Set type to object. See https://github.com/CMSgov/cdap/pull/268/files#diff-8e04d31622ba5ffa53a573b60074ebc6a35c310f36c4bd28c5f3857ad9da571fR32